graphql/type.go

```diff
@@ -33,3 +33,14 @@ const (
 func (t *Type) IsInputObject() bool {
 	return t.Kind == TypeKindInputObject
 }
+
+// GetErrorField return the error field if there is one
+func (t *Type) GetErrorField() *Field {
```

Comment:
nit: I think go conventions say this should just be `ErrorField()`

Reply:
Fixed